cmd/centrifuge/root.go

```diff
@@ -39,6 +41,12 @@ func Execute() {
 	}
 }
 
+func printStartMessage() {
+	c := figure.NewFigure("Centrifuge OS", "", true)
```

Comment:
Do you have a screenshot you can attach of how it would look like?

Reply:
<img width="664" alt="Screenshot 2019-04-10 at 11 30 17" src="https://user-images.githubusercontent.com/654927/55868087-0dfce900-5b84-11e9-936a-44b0d9688f5b.png">